cirq/ops/pauli_string.py

```diff
@@ -164,6 +164,16 @@ def __mul__(self, other) -> 'PauliString':
                            qubit_pauli_map=self._qubit_pauli_map,
                            coefficient=self.coefficient)
 
+    @property
+    def gate(self) -> 'cirq.DensePauliString':
+        order: List[Optional[pauli_gates.Pauli]] = [
+            None, pauli_gates.X, pauli_gates.Z, pauli_gates.Y
```

Comment:
You're generally not supposed to have mutable default function arguments (like list)

Reply:
That's not an argument, it's a local variable with a type declaration.

Reply:
ah, my brain got confused

Reply:
this type annotation seems to hurt rather than help readability, imo